cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/dataset/SortInfo.java

```diff
@@ -46,7 +46,8 @@
   public enum SortOrder {
     ASC,
     DESC,
-    WEIGHTED
+    WEIGHTED,
+    UNSORTED
```

Comment:
Is it fine to re-use the SortOrder enum? Or should we create one separately for Pagination?

Reply:
We don't need unsorted as this will not work for pagination. Pagination always requires some kind of sorting